### ISSUE TITLE
GridView, can you help to provide freeze pane function?

### ISSUE THREAD

**Issue by ScottHuangZL** [yiisoft/yii2#8770]
Dear Yii2 Team,

Some grid widget can provide freeze top header function. However, I do not find any Yii2 grid can help to freeze left columns function.

Do you have a plan to enable GridView to provide function to freeze first several columns when scroll to right, just like excel freeze pane? thanks.

Regards,
Scott Huang

**Comment by samdark** [comment]
No, we don't have such plan but could consider it if pull request provided.

**Comment by ScottHuangZL** [comment]
Any progress?  Or new plan?
It is quite common in some React/Angular/Vue grid components. Even in Java Vadin framework.

Thanks.

**Comment by ScottHuangZL** [comment]
As you know what Yii attracting people, one of the reason is its out of box widgets, including this powerful grid.  Appreciate add this feature ASAP. 

thanks.
